### PR TITLE
human-readable client-side network stats logging

### PR DESCRIPTION
**Pull request by dshpet**
# Description

Make network logging more human-readable and understandable.

- Add tabbed alignment;
- Use Megabytes and Percent as metric
- Visually indent stats message.

## Type of change

Minor visual quality-of-life improvement for end-user

## How Has This Been Tested?

Tested on one local development machine.

WTR:
1. Run the app. 
2. Wait for usual message update timing after attack targets were written down.
3. Look at the terminal.

## Test Configuration

- Release version: [937ec0f](https://github.com/Arriven/db1000n/commit/937ec0fa1055c38d2f97436359f752fa316e33aa) 
- Platform: Windows x64

## Logs

```
!Атака проводиться успішно! Руський воєнний корабль іди нахуй!
!Attack is successful! Russian warship, go fuck yourself!
---------Traffic stats---------
 [     Generated ] 0.70 MB | 733382 bytes
 [      Received ] 0.06 MB |  67398 bytes
 [ Response rate ]  9.2%
-------------------------------
```

## Screenshots
How it looks in Windows PowerShell
image

### CODE REVIEW

**Review by Arriven — APPROVED** [review]

**Comment by Arriven** [comment]
@dshpet lint is failing for your PR

File: src/runner/runner.go [hunk 1]
```diff
@@ -155,14 +158,26 @@ func dumpMetrics(clientID string, debug bool) {
...
-		log.Printf("The app has generated approximately %v bytes of traffic\n", bytesGenerated)
+		fmt.Println()
 
-		if bytesProcessed > 0 {
-			log.Printf("Of which for %v bytes we received some response from the target", bytesProcessed)
-		}
+		fmt.Println("Атака проводиться успішно! Руський воєнний корабль іди нахуй!")
+		fmt.Println("Attack is successful! Russian warship, go fuck yourself!")
+
+		const BytesInMegabytes = 1024
```
Comment (Lagovas) [inline src/runner/runner.go:166]: actually 1024*1024 bytes in one megabyte... 1024 is one kilobyte.

Comment (dshpet) [reply src/runner/runner.go]: Fixed. 
Guess not the best idea to commit at night.

**Comment by dshpet** [comment]
Fixed linter errors.
Improved formatting even further.

**Review by Arriven — APPROVED** [review]

**Pull request merged**